models/PostImport.php

```diff
@@ -45,6 +45,12 @@ public function importData($results, $sessionKey = null)
             throw new ApplicationException('Please specify a match for the Categories column.');
         }
 
+        $except = ['id', 'categories', 'author_email'];
+        $except_override = $this->fireEvent('winter.blog.import.except', [$except], true);
+        if ($except_override) {
+            $except = $except_override;
+        }
```

Comment:
@mjauvin what would the use be for this event, over just not linking the columns from the CSV to a corresponding field in the model?

Reply:
@bennothommo if you look at my example in the PR description, that should be obvious. Basically, you want to ignore the field in the loop that assigns its value, but still do some processing on it afterward (using the `winter.blog.import.afterSave` event).

Reply:
@mjauvin I'm just trying to ascertain other uses for having a combination of events like this. The two of them seem specific just for your use case, and as you mentioned, there's no use for the `except` event except if the data is to be used in the second event.

Could we not perhaps have a single event before [this line](https://github.com/wintercms/wn-blog-plugin/blob/main/models/PostImport.php#L75) that could act as a `beforeSave` that could populate special fields as necessary and then remove them from the data before passing it on to the rest of the original functionality?

Reply:
@bennothommo The `except` event can also be used to remove fields like "id" which can be useful if you want to recreate the Posts with the original IDs (to make it easier to reference the posts in other places).

So unless that other `single` event also gets a reference to the $except array, that would prevent this.